paddle/fluid/framework/executor.cc

```diff
@@ -168,8 +170,8 @@ void Executor::Run(const ProgramDesc& pdesc, Scope* scope, int block_id,
                    const std::vector<std::string>& skip_ref_cnt_vars,
                    bool force_disable_gc, bool keep_kid_scopes) {
   platform::RecordBlock b(block_id);
-  if (FLAGS_use_mkldnn) EnableMKLDNN(pdesc);
 #ifdef PADDLE_WITH_MKLDNN
+  if (FLAGS_use_mkldnn) EnableMKLDNN(pdesc);
```

Comment:
If you're putting `EnableMKLDNN` under `#ifdef` then you can remove the `ifdef`s from this function https://github.com/PaddlePaddle/Paddle/blob/ec2b0e14909cec83797da434e9cfbff27d11dc20/paddle/fluid/framework/executor.cc#L577